packages/svelte-ux/src/lib/components/DateField.svelte

```diff
@@ -65,6 +78,15 @@
     inputValue = undefined;
     dispatch('change', { value });
   }}
+  classes={{
+    root: cls('DateField', settingsClasses.root, classes.root, $$props.class),
+    container: cls(settingsClasses.container, classes.container),
+    label: cls(settingsClasses.label, classes.label),
+    input: cls(settingsClasses.input, classes.input),
+    error: cls(settingsClasses.error, classes.error),
+    prepend: cls(settingsClasses.prepend, classes.prepend),
+    append: cls(settingsClasses.append, classes.append),
+  }}
```

Comment:
btw with something like a `clsMulti` (bikeshed that some hah), this could be reduced to…

```jsx
classes={clsMulti(
  settingsClasses,
  classes,
  { root: cls('DateField', $$props.class) }
)}
```